src/Polly.Core/Retry/RetryConstants.cs

```diff
@@ -0,0 +1,24 @@
+namespace Polly.Retry;
+
+internal static class RetryConstants
+{
+    public const string StrategyType = "Retry";
+
+    public const string OnRetryEvent = "OnRetry";
+
+    public const RetryBackoffType DefaultBackoffType = RetryBackoffType.Exponential;
+
+    public const int DefaultRetryCount = 3;
+
+    /// <summary>
+    /// Maximal allowed retry counts unless infinite.
```

Comment:
```suggestion
    /// Maximum allowed retry count, unless infinite.
```

Reply:
This was internal so I just dropped comments.